spanner/doc.go

```diff
@@ -338,10 +338,14 @@ NULL-able STRUCT values.
 # DML and Partitioned DML
 
 Spanner supports DML statements like INSERT, UPDATE and DELETE. Use
-ReadWriteTransaction.Update to run DML statements. It returns the number of rows
-affected. (You can call use ReadWriteTransaction.Query with a DML statement. The
-first call to Next on the resulting RowIterator will return iterator.Done, and
-the RowCount field of the iterator will hold the number of affected rows.)
+ReadWriteTransaction.Update to run simple DML statements and
+ReadWriteTransaction.Query to run DML statements containing "THEN RETURN"
```

Comment:
I think mentioning "THEN RETURN" here will make it specific to GoogleStandardSQL dialect (PostgreSql dialect uses "RETURNING"). Should we change the documentation to reflect both the dialects?

Reply:
Done.

Reply:
@c2nes: Can you check if we missed the documentation update here?

Reply:
Whoops, sorry about that. Forgot to stage the doc update before amending. Should be updated now.